content/en/docs/Concepts/MVC.md

```diff
@@ -93,11 +93,11 @@ public ActionResult Index()
 }
 ```
 
-Now, when /Index is referenced in the URL, the program will return the corresponding view.
+Now, when "/Index" is referenced in the URL, the program will return the corresponding view.
```

Comment:
My apologies from last time, instead of quotes wrap it in single backticks for an inline code block. `/Index`